build-configuration/pom.xml

```diff
@@ -2,12 +2,6 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>
 
-   <parent>
-      <groupId>org.jboss</groupId>
-      <artifactId>jboss-parent</artifactId>
-      <version>36</version>
-   </parent>
```

Comment:
The `jboss-parent` is mandatory downstream. Please keep it upstream so we can detect issues sooner than later.

Reply:
I'm sorry. I had `http://repository.jboss.org/nexus/content/groups/public/` instead of `https://repository.jboss.org/nexus/content/groups/public-jboss/ ` as `jboss-public-repository-group` url in settings.xml. Now it works. I'll drop this commit.

Reply:
for the record, you can add the option `-Dinsecure.repositories=WARN` to your `mvn` command if you had to use an insecure maven repository (i.e. non https).